status/projects.py

```diff
@@ -1059,87 +1057,89 @@ class PrioProjectsTableHandler(SafeHandler):
     Loaded through /api/v1/prio_projects"""
 
     def get(self):
-        self.set_header("Content-type", "application/json")
-        projects = OrderedDict()
-        def_dates_gen = { 'days_recep_ctrl' : ['open_date', 'queued'],
-                        'days_analysis' : ['all_samples_sequenced', 'best_practice_analysis_completed'],
-                        'days_data_delivery' : ['best_practice_analysis_completed', 'all_raw_data_delivered'],
-                        'days_close' : ['all_raw_data_delivered', 'close_date']
-                        }
-
-        def_dates_summary = { 'days_prep_start' : ['queued', 'library_prep_start'],
-                            'days_seq_start' : [['qc_library_finished', 'queued'], 'sequencing_start_date'],
-                            'days_seq' : ['sequencing_start_date', 'all_samples_sequenced'],
-                            'days_prep' : ['library_prep_start', 'qc_library_finished']
-                            }
-
+        from collections.abc import Iterable
+        def flatten(l):
+            for el in l:
+                if isinstance(el, Iterable) and not isinstance(el, str):
+                    yield from flatten(el)
+                else:
+                    yield el
```

Comment:
Good idea to add a function for this. I would never have thought of this code myself, but it looks neat! 👏